python/utils.py

```diff
@@ -784,22 +787,23 @@ def write_airr_output(outfname, annotation_list, cpath, failed_queries, debug=Fa
                 writer.writerow({'sequence_id' : failfo['unique_ids'][0], 'sequence' : failfo['input_seqs'][0]})
 
 # ----------------------------------------------------------------------------------------
-def get_linearham_line(lh_line, seqfos, glfo, debug=False):
-    line = {}
-    for lhk in lh_line:  # [k for k in lh_line if k in linearham_headers and linearham_headers[k] is not None]:  # limit to the ones with a direct partis correspondence
-        if lhk not in linearham_headers or linearham_headers[lhk] is None:
+def process_input_linearham_line(lh_line):
+    """ convert <lh_line> (see linearham_headers). Modifies line. """
+    for lhk in set(lh_line):  
```

Comment:
I don't think I understand what the set() is for -- dict keys already have to be unique, right?

Reply:
@psathyrella since we modify the dict keys while iterating. Is this bad style / bad hack?

Reply:
oh, duh, right, no i just didn't notice. Good to add the comment though.